proxy/src/mgmt.rs

```diff
@@ -95,7 +95,7 @@ impl postgres_backend::Handler for MgmtHandler {
 }
 
 fn try_process_query(pgb: &mut PostgresBackend, query_string: &str) -> anyhow::Result<()> {
-    println!("Got mgmt query: '{}'", query_string);
+    println!("Got mgmt query [redacted]"); // Content contains password, don't print it
```

Comment:
Maybe we could use `cfg(debug_assertions)` to print sensitive info only in debug mode. It's not like we'd use debug builds in prod environment, so this seems to be an appropriate measure.

Reply:
Good idea. I don't see the need for this extra complexity today, but if the need arises it's a valid option. Staying on the safe side for now.